python/ray/rllib/models/action_dist.py

```diff
@@ -5,6 +5,7 @@
 from collections import namedtuple
 import distutils.version
 import tensorflow as tf
+import torch
```

Comment:
Let's make sure to not import torch unless we hit a torch=true code path, to avoid acquiring a hard dependency on torch.